source/elements/oneDAL/source/programming_model/algorithm_anatomy.rst

```diff
@@ -168,6 +179,28 @@ refers to one of the computational methods as shown in the example below.
    } // namespace oneapi::dal::%ALGORITHM%
 
 
+.. _tasks:
+
+Computational Tasks
+-------------------
+The supported computational tasks are declared within the ``%ALGORITHM%::task``
+namespace using tag-types. Algorithm shall support at least one task and declare
+the ``by_default`` type alias that refers to one of the tasks as shown in the
```

Comment:
Somewhere, in the section or earlier we need a sentence that explains specifics of the task in cases such as covariance